pages/index.vue

```diff
@@ -134,6 +134,11 @@ export default {
               }
               .output-logo {
                 margin-bottom: 50px;
+                background-color: #ffffff;
+              }
+              .output-text {
+                margin-bottom: 50px;
```

Comment:
is this intended?

Reply:
No, sorry my bad.

Reply:
My actual though was to wrap text instead of just a margin. My new commit fixes it.